src/pages/index.js

```diff
@@ -91,53 +75,106 @@ function renderCard(data) {
 const imagePopup = new PopupWithImage("#preview-image-modal");
 imagePopup.setEventListeners();
 
-//FORM POPUP
+//DELETE CARD POPUP
+const deleteCardPopup = new PopupWithConfirmation("#card__delete-modal");
+deleteCardPopup.setEventListeners();
+
+const profileUpdateSubmit = new PopupWithConfirmation("#profile-edit-modal");
```

Comment:
`PopupWithConfirmation` is only for confirming rather than adding new cards or submitting forms